swingset/src/main/java/com/nqadmin/swingset/SSDBNavImpl.java

```diff
@@ -147,6 +147,9 @@ protected void setComponents(final Container _container) {
             } else if(comps[i] instanceof JTextArea) {
             // IF IT IS A JTextArea, SET TO EMPTY STRING
                 ((JTextArea)comps[i]).setText("");
+            } else if (comps[i] instanceof SSBaseComboBox) {
```

Comment:
Looks like setSelectionPending() takes care of setting to nullItem so setSelectionPending(true) is sufficient here.

Reply:
Right, at least that's the ide.